Package/PackageActions/Install.cs

```diff
@@ -151,12 +152,39 @@ private int DoExecute(CancellationToken cancellationToken)
                 bool askToInstallDependencies = !NonInteractive;
                 if (Force)
                     askToInstallDependencies = false;
-                List<PackageDef> packagesToInstall = PackageActionHelpers.GatherPackagesAndDependencyDefs(
-                    targetInstallation, PackageReferences, Packages, Version, Architecture, OS, repositories, Force,
-                    InstallDependencies, Force, askToInstallDependencies, NoDowngrade);
 
-        
-                
+                List<PackageDef> packagesToInstall;
+                try
+                {
+                    packagesToInstall = PackageActionHelpers.GatherPackagesAndDependencyDefs(
+                        targetInstallation, PackageReferences, Packages, Version, Architecture, OS, repositories, Force,
+                        InstallDependencies, Force, askToInstallDependencies, NoDowngrade);
+                }
+                catch (ImageResolveException ex)
+                {
+                    // If the problem is exactly that we failed to resolve a compatible release version of a package,
+                    // ask the user to retry the resolution with a pre-release instead.
+                    // We could also ask for an RC / Any, but if there are no releases, then it is likely also the case that there are no RCs.
+                    // 'any' would be the most likely specifier to succeed, but it is probably a bit too extreme to suggest something so bleeding edge.
+                    // A beta version strikes a good middle ground. It is pretty likely to resolve, and probably won't be too unstable.
+                    if (NonInteractiveUserInputInterface.IsSet()) throw;
+                    if (false == (ex.Result is FailedImageResolution fir)) throw;
+                    if (fir.resolveProblems.Count != 1) throw;
+
+                    var problem = fir.resolveProblems[0];
```

Comment:
Should we make sure the problem is with the package the user is trying to install? We could get problems with other packages, right?